dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -1136,6 +1136,17 @@ def assert_caching_enabled(cache_control_header, max_age, proxy_max_age)
     assert_select 'button', I18n.t('teacher.panel.example')
   end
 
+  test "can view CSF teacher markdown as non-authorized teacher" do
+    refute can_view_teacher_markdown?
+
+    stubs(:current_user).returns(@teacher)
+    @script.stubs(:k5_course?).returns(true)
+    assert can_view_teacher_markdown?
+
+    stubs(:current_user).returns(@student)
+    refute can_view_teacher_markdown?
+  end
```

Comment:
That standalone test would be testing that users who aren't logged in can't see teacher markdown. If I make a change to `can_view_teacher_markdown?` that screws up the logic for logged-out users, but it still works correctly for non-authorized teachers, it would be nice if this test case kept passing but a separate `logged out users can't see teacher markdown` test failed.